org.clementine_player.Clementine.yaml

```diff
@@ -0,0 +1,262 @@
+app-id: org.clementine_player.Clementine
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: clementine
+rename-desktop-file: clementine.desktop
+rename-icon: clementine
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
```

Comment:
What does it need `all` for?

Reply:
I assume this is needed for the iPhone support, but have no way of actually testing.

Reply:
Then remove it, no point in keeping it if it's not tested.

Reply:
Untested libraries removed and `--device` set to `dri`.